### ISSUE TITLE
py3dtilers is not referenced by google

### ISSUE THREAD

**Issue by jailln** [VCityTeam/py3dtilers#70]
It's maybe a little bit secondary but py3dtilers is not referenced by google: searching py3dtilers in google does not yield any result for this repository on none of the two pages. It yields results for py3dtiles, or for py3dtilers on githubmemory, but not the github repository. This may have a strong impact on its visibility.

**Comment by LorenzoMarnat** [comment]
Indeed, and it makes me very sad. Maybe @EricBoix knows some (free) magic tricks to reference it ?

**Comment by EricBoix** [comment]
Yep, it seems that for some undocumented reason google's crawler didn't reach py3dtileRs yet. 
And it seems that [google indexing submission process](https://developers.google.com/search/docs/advanced/crawling/ask-google-to-recrawl?utm_source=wmx&utm_medium=deprecation-pane&utm_content=submit-url)  requires you to patch the html file that is designated/pointed by the URL you submit for indexing. But alas git repositories generally don't include html files but only source code or markdown files that cannot be patched with google verification tag.
(we could place google's verification html file in the git repository but this is quite a "noisy/kludgy" way of doing things)

Some stackoverflow topics (e.g. [here](https://stackoverflow.com/questions/26199705/github-repository-not-listing-in-google-search) or [there](https://stackoverflow.com/questions/55827354/github-repository-not-listing-in-google-search-no-way-to-submit-url)) suggest to activate the github-pages associated with the repository one wishes to get indexed, in order to include the verification code there (because html files get served) and to refer to the original source repository... Sigh.

Anyhow, even without such side-effect tricks, some of VCityTeam repositories on github already got indexed by the google crawler (like [UD-SV](https://www.google.com/search?q=ud-sv)). And because, in the end, the probability for a crawler to find a page is probably correlated with the number of pages referencing it, I updated e.g [UD-SV component page](https://github.com/VCityTeam/UD-SV/blob/master/SoftwareComponents/Readme.md#backend) to refer to py3dTileRs.

Wait and see ? 

Fun note: funnily enough duckduckgo [already indexes py3dTileRs](https://duckduckgo.com/?q=py3dtilers).
